plaidml/op/op_test.cc

```diff
@@ -176,6 +176,71 @@ TEST_F(OpTest, Concatenate) {
   runProgram(program);
 }
 
+TEST_F(OpTest, 3dConvolution) {
```

Comment:
@tzerrell @jbruestle  This is actually the really slow test case. All the rest of these are passing for me, but this one runs for several minutes. I haven't profiled yet, but my guess is that it's either BILP or XSMM stenciling.